### ISSUE TITLE
Parallel test execution of Spek tests via Gradle

### ISSUE THREAD

**Issue by schalkms** [JetBrains/spek#716]
To cite the [Gradle documentation](https://guides.gradle.org/performance/#parallel_test_execution)

> Gradle will happily run multiple test cases in parallel, which is useful when you have several CPU cores and don’t want to waste most of them. 

```Kotlin
tasks.withType<Test> {
    maxParallelForks = 4
}
```

Can we also have support for `Spek` to speed up test execution?

**Comment by raniejade** [comment]
Duplicate of #108. Definitely on the roadmap, but no concrete plans yet.
